src/Web/MASA.PM.Web.Admin/Pages/Home/AppModal.razor

```diff
@@ -1,6 +1,6 @@
 ﻿@inherits PmCompontentBase
 
-<SSheetDialog @key="@($"app-{_appFormModel.Visible}")" ValueChanged="AppModalValueChanged" Value="_appFormModel.Visible" Title="@(_appFormModel.HasValue ? T("Edit App"):T("Add App"))" Icon="mdi-circle" IconColor="success">
+<SSheetDialog ValueChanged="AppModalValueChanged" Value="_appFormModel.Visible" Title="@(_appFormModel.HasValue ? T("Edit App"):T("Add App"))" Icon="mdi-circle" IconColor="success">
```

Comment:
Add 'EnableDomReload' prop

Reply:
The latest main branch code has been merged，now use SSimpleModal, without 'EnableDomReload'